.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+# Java Maven CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8-jdk
+    working_directory: ~/repo
+    environment:
+      MAVEN_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - run: git submodule sync
+      - run: git submodule update --init
+
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "pom.xml" }}
+
+      - run:
+          name: Install dependencies
+          command: mvn dependency:go-offline
+
+      - save_cache:
+          paths:
+            - ~/.m2
+          key: v1-dependencies-{{ checksum "pom.xml" }}
+
+      - run:
+          name: Run tests
+          command: mvn integration-test
+          
+      - run:
```

Comment:
Could this be it's own job? Is it dependent on tests?

Reply:
This runs all the stuff needed for release without pushing it to the remote repository or uploading the binaries to bintray. The last release that was made there was a mess up and the release was not completely properly and we had to get CircleCI to intervene to fix the issue. This just insures that it is very possible to make a release from each revision that is made.

Reply:
I understand what it's for, but it is being run sequentially. Doesn't appear to be directly dependent on tests?

Reply:
It is an independent step from tests, but considering our recent discussion a bunch of things are going to change.